### ISSUE TITLE
Frontend should not ignore error_key for single rule endpoint

### ISSUE THREAD

**Issue by Serhii1011010** [RedHatInsights/insights-results-smart-proxy#210]

**Comment by bond95** [comment]
@Sergey1011010 Why do we need to ignore `error_key`?

**Comment by Sergey1011010** [comment]
@bond95 we don't, as the title says. The was the issue that it was ignored, but I guess it could be fixed with some of the refactorings. We just need to make sure that it works as intended(single rule endpoint uses both rule id and error key to find a rule).

**Comment by bond95** [comment]
@Sergey1011010 yep, my bad. But I still don't get what is issue about, because `error_key` is not ignored.

**Comment by Sergey1011010** [comment]
@bond95 well, the issue was there, maybe not for frontend, don't remember, but definitely it was ignored on the backend side.

**Comment by Sergey1011010** [comment]
@bond95 if the frontend does provide rule_id and error_key to smart proxy's endpoint, we can close this one

**Comment by bond95** [comment]
@Sergey1011010 ok, then I'll close it.

**Comment by bond95** [comment]
Reopen it, as frontend, don't provide error_key
